singularity-opac/chebyshev/vandermonde.hpp

```diff
@@ -823,6 +800,45 @@ constexpr Real Vandermonde21[21][21] = {
      -0.02119246990060137183703833947588521518727195892995666241,
      0.00711715177013564326580378530807301460700464334926828038}};
 
+  PORTABLE_INLINE_FUNCTION void get_vmbox(Real *r)
+  {
+      constexpr Real Vandermonde9[9][9] = {
+        {0.11111111111111, 0.11111111111111, 0.11111111111111, 0.11111111111111,
```

Comment:
Thanks for the fix. Glad this works. I don't think this solution is *too* awful with a little cleanup. Of course a. refactor where the generator code is in the source would be better.

Reply:
I think it would make more sense for an object to generate these at runtime during startup.

Reply:
Yes I agree, but this is a step in that direction.